example/package.json

```diff
@@ -25,5 +25,6 @@
     "files": [
       "test/*.test.js"
     ]
-  }
+  },
+  "browserslist": ["last 1 Chrome versions"]
```

Comment:
This is only for tests at the moment? Do we want to include other browsers in the future?

Reply:
Yes, the target for this build is mainly puppeteer. I added a comment to explain this. Without this, Parcel bundles it in a way which requires `regeneratorRuntime`. Context: https://github.com/parcel-bundler/parcel/issues/1762#issuecomment-504438377